applications/nrf5340_audio/src/modules/lc3_playback.c

```diff
@@ -0,0 +1,228 @@
+#include "lc3_playback.h"
+#include <stdint.h>
+#include <zephyr/sys/ring_buffer.h>
+#include "sd_card.h"
+#include "hw_codec.h"
+#include "sw_codec_lc3.h"
+#include "sw_codec_select.h"
+#include "pcm_stream_channel_modifier.h"
+#include "pcm_mix.h"
+#include "audio_i2s.h"
+#include "sw_codec_lc3.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(lc3_playback, 4); /* For draft PR feedback: going to change this line */
+
+#define RING_BUF_SIZE		1920 /* This can be modified */
+#define lc3_playback_STACK_SIZE 4096
+
+/*File structure of the LC3 encoded files*/
+struct lc3BinaryHdr_t {
+	uint16_t fileId;		/* Constant value, 0xCC1C */
+	uint16_t hdrSize;		/* Header size, 0x0012 */
+	uint16_t sampleRate_divided100; /* Sample frequency / 100 */
+	uint16_t bitRate_divided100;	/* Bit rate / 100 (total for all channels) */
+	uint16_t channels;		/* Number of channels */
+	uint16_t frameMs_times100;	/* Frame duration in ms * 100 */
+	uint16_t rfu;			/* RFU value */
+	uint16_t signalLen;		/* Number of samples in signal, 16 LSB */
+	uint16_t signalLenRed;		/* Number of samples in signal, 16 MSB (>> 16) */
+};
+
+static struct lc3BinaryHdr_t header;
```

Comment:
Would be great if we add something for indicating "header" is for encoded file, also for header_size.